src/Tickets/Commerce/Ticket.php

```diff
@@ -431,26 +452,17 @@ public function get_ticket( $ticket_id ) {
 	 * @todo  TribeCommerceLegacy: Move this method into the another place.
 	 *
 	 * @since 5.1.9
+	 * @deprecated TBD This method is deprecated due to the Legacy Tribe Commerce no longer being supported.
 	 *
 	 * @param int $ticket_id The ticket post ID.
 	 *
 	 * @return int
 	 */
 	protected function get_cancelled( $ticket_id ) {
-		$denied_orders = \Tribe__Tickets__Commerce__PayPal__Order::find_by( array(
-			'ticket_id'      => $ticket_id,
-			'post_status'    => Denied::SLUG,
-			'posts_per_page' => - 1,
-		), [
-			'items',
-		] );
-
-		$denied = 0;
-		foreach ( $denied_orders as $denied_order ) {
-			$denied += $denied_order->get_item_quantity( $ticket_id );
-		}
```

Comment:
We still need to fetch this data, even if the existing one was fetched incorrectly. The method is not deprecated, we just need to fix it.